src_assets/common/assets/web/utils/agents/core/agentCore.js

```
@@ -0,0 +1,179 @@
+
+  return {
+    ...definition,
+    id,
+    type: definition.type || options.defaultType || 'extension',
+    label: definition.label || id,
+  }
+}
+
+export function createCapabilityFromSkill(skill, capability = {}) {
+  return {
+    skillId: skill.id,
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)